README.md

```diff
@@ -122,6 +122,9 @@ module "github-actions-runner-aws" {
   environment               = "${environment}"
   ecs_vpc_id                = "${vpc.id}"
   ecs_subnet_ids            = "${vpc.private_subnets.id}"
+  assign_public_ip     = // optional, defaults to false
```

Comment:
This should follow the same documentation convention as other optional parameters (see towards the bottom of the readme).